### PR TITLE
[sdk] Return subgraph query; tidy up method headers

### PR DESCRIPTION
**Pull request by jakekidd**
## The Problem

- We weren't actually returning the query result in the 'generic' subgraph query before 🤦‍♂️ 

## The Solution

- Now we do

- Also did a bit of tidying up, the inconsistent fn headers were bothering me.

## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [x] Update documentation if needed.
- [x] Update CHANGELOG.md.

### CODE REVIEW

**Review by LayneHaber — CHANGES_REQUESTED** [review]
Changelog and tests

File: packages/sdk/src/sdk.ts [hunk 1]
```diff
@@ -309,10 +308,6 @@ export class NxtpSdk {
 
     const signerAddr = await this.config.signer.getAddress();
     const connectedSigner = this.config.signer;
-    // TODO: Safe to remove?
-    if (isNode()) {
-      // connectedSigner = this.config.signer.connect(this.config.chainConfig[sendingChainId].providers[0]);
-    }
```
Comment (LayneHaber) [inline packages/sdk/src/sdk.ts:315]: who commented this out instead of just deleting it lol

File: packages/sdk/src/sdk.ts [hunk 2]
```diff
@@ -475,7 +466,7 @@ export class NxtpSdk {
   }
 
   /**
-   * Cancels the given transaction
+   * Cancels the given transaction.
```
Comment (LayneHaber) [inline packages/sdk/src/sdk.ts:469]: 🏅

**Review by LayneHaber — APPROVED** [review]
Nice job

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**